src/services/fileStorage/proxy-service.js

```diff
@@ -11,27 +11,34 @@ const {
 	canWrite,
 	canRead,
 	canCreate,
-	canDelete,
+	canDelete
 } = require('./utils/filePermissionHelper');
-const { returnFileType, generateFileNameSuffix: generateFlatFileName } = require('./utils/filePathHelper');
+const {
+	returnFileType,
+	generateFileNameSuffix: generateFlatFileName
+} = require('./utils/filePathHelper');
 const { FileModel } = require('./model');
 const RoleModel = require('../role/model');
 const { courseModel } = require('../user-group/model');
 const { teamsModel } = require('../teams/model');
 const { sortRoles } = require('../role/utils/rolesHelper');
 const { userModel } = require('../user/model');
 
+const FILEPREVIEW_SERVICE_URI = 'http://localhost:3000/filepreview';
```

Comment:
Please move to an ENV

Reply:
use `= ENV || 'http://localhost:3000/filepreview'`